docs/src/nav.ts

```diff
@@ -88,6 +88,10 @@ export const nav = [
         label: 'Menu',
         href: '/react/components/menu',
       },
+      {
+        label: 'Menubar',
```

Comment:
Nitpick: What is that rational for `Menubar` vs `Menu Bar` here?
To me the component name is similar to `Preview Card` more than `Fieldset` (which is the only component with 2 words that we don't split in the navbar AFAIK).

Reply:
Mostly just adhering to ARIA. They do "Menubar", "Toolbar", "Alert Dialog", "Fieldset".

I think the rationale is that "Bar" is not a thing on its own. Whereas "Dialog" is a thing.

Preview Card is not an ARIA pattern, we just made it up. But "Card" would be a thing on its own, so it follows the same logic.

Reply:
OK, the naming does not feel super natural to me, but adhering to ARIA is a good reason to stick with it